docs/GOVERNANCE.md

```diff
@@ -0,0 +1,155 @@
+- Act as a communication bridge between the wider community and the developers.
+- Be available for communication from all community members.
+- Suggest granting of roles to community members.
+- Encourage positive community interactions and growth.
+
+**Notes**
+- A community organizer is not the same thing as a moderator. A moderator's role is in enforcement, while a community organizer's role is in analysis and interaction.
+
+### Oathkeeper (Moderator)
+GitHub Permissions: Moderator
+
+Description: Moderators help to keep discussions civil, ensure disagreements remain productive, de-escalate conflict, resolve communication mishaps, and prioritize creating a friendly and welcoming environment that assures contributors their time will be appreciated and valued.
+
+**Responsibilities/Duties:**
+- Show a genuine desire to foster a welcoming environment.
+- Stay cool, calm, and collected even in tense situations.
+- Display good communication and conflict-resolution skills.
+- Show that they can be trusted to moderate fairly and unbiasedly.
+- Show a willingness to resolve conflicts even with uncooperative parties.
+
+**Notes:**
+- While our moderation team is equipped with the tools to take moderation actions when necessary, we believe that this should be considered a last resort option. Our moderation team prefers to de-escalate and resolve conflicts and only utilizes moderation actions such as verbal warnings, time-outs, or more if de-escalation is impossible.
+
+### Organization Member
+GitHub Permissions: Read
+
+Description: Members are individuals who have been added to the Endless Sky organization without being granted a role. This is typically so they can be requested for reviews or given repository-specific permissions for specific projects.
+
+**Responsibilities/Duties:**
+Address the reviews of reviewers and developers when requested.
+
+**Notes**
+- While members do not have any organization-level permissions they may have repository-level permissions if necessary for their role.
+- Members are typically individuals who have contributed a great deal to a specific faction and are often requested for reviews on that faction.
+
+### Contributor
+GitHub Permissions: Read
+
+Description: Contributors are those who create the majority of the pull requests for the game. This is not a role that is given to individuals, but one that someone steps into in deciding to open a pull request.
+
+**Responsibilities/Duties:**
+Address the reviews of reviewers and developers on their pull requests.
+
+**Notes:**
+- Includes "content creators," which is a more specific term for contributors who open content pull requests.
+- Contributors need not blindly accept every suggestion from a reviewer. If a contributor disagrees with a review comment, then they should explain why they disagree, particularly if the changes being requested are substantial. See the "On Review Relationships" and "On Reviews and Approvals" sections for more detail.
```

Comment:
I can't seem to find where the `"On Review Relationships"` and `"On Reviews and Approvals"` sections are -- should these link to somewhere?

Reply:
Ah, yes. That's from the Development Manifesto I copied this section from. 
I removed most of the references to other sections but must have missed this one. Thanks for pointing it out.